gvsbuild/patches/hello-world/meson.build

```diff
@@ -0,0 +1,2 @@
+project('tutorial', 'c')
```

Comment:
nit++: `project('hello-world', 'c')` for consistency

Reply:
Updated!